### DISCUSSION TITLE
How to load a plugin with a `<script />` tag?

### DISCUSSION THREAD

**Discussion by UndarkAido** [pixijs/pixijs#8973] [Q&A] [answered]
I'm currently using PIXI as such:
```js
<script type="text/javascript" src="/lib/pixi.js"></script>
```

And I'd like to install the Extract plugin. Where can I find a download? When I have it should it be included before or after PIXI? Thanks for the great project! I've been enjoying using it.

**Comment by SuperSodaSea** [comment, marked as answer]
`@pixi/extract` is bundled within the `pixi.js` bundle, so you probably don't need to install it manually. You can access it with `app.renderer.extract` (PixiJS v7) or `app.renderer.plugins.extract` (PixiJS v6).

**Reply by UndarkAido** [reply]
That worked! Now it throws this when I call it with a texture though:
```
Uncaught TypeError: displayObject.getLocalBounds is not a function
    at GenerateTextureSystem.generateTexture (pixi.js:11162:52)
    at _Renderer.generateTexture (pixi.js:11634:36)
    at _Extract.pixels (pixi.js:18112:41)
    at textureToMatrix (index.html?_ijt=n0ug69ad1phesdp7cp04b22mqn:61:46)
    at HTMLDocument.<anonymous> (index.html?_ijt=n0ug69ad1phesdp7cp04b22mqn:557:29)
```

**Reply by SuperSodaSea** [reply]
`extract.pixels()`'s argument should be `DisplayObject | RenderTexture`. A normal `Texture` is not supported.

**Reply by UndarkAido** [reply]
Putting it in a `Sprite` worked, thank you!
